competitions/template/ansible/inv/prod/group_vars/all/base

```diff
@@ -0,0 +1,15 @@
+---
+competition_name: <COMPETITIONNAME>
+db_username: <COMPETITIONNAME>_rw
+torquedataconnect_sheet_name: <COMPETITIONNAME>
```

Comment:
Same thing about "'`<USUALLY_COMPETITION_NAME>`" for this one, so it more clearly correlates with the value from `all.tmpl`.